include/private/config/locale/da-dk.h

```diff
@@ -0,0 +1,257 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/*
+ * Copyright 2020-2023 Joel E. Anderson
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @file
+ * United States English localization strings.
```

Comment:
Update this to describe your danish translation.

Reply:
unsure what you mean by this

Reply:
Instead of the `@file` directive saying "United States English localization strings", it should say something like "Danish localization strings".